VirtualReality/Widgets/qMRMLVirtualRealityView.cxx

```diff
@@ -93,8 +96,12 @@ qMRMLVirtualRealityViewPrivate::qMRMLVirtualRealityViewPrivate(qMRMLVirtualReali
   , CamerasLogic(NULL)
 {
   this->MRMLVirtualRealityViewNode = 0;
+  this->HomeWidget = new qMRMLVirtualRealityHomeWidget(q_ptr);
+  
```

Comment:
You're adding too much unnecessary whitespace that makes the code harder to read